crypto/ed25519/math/point.go

```diff
@@ -0,0 +1,92 @@
+package math
+
+import (
+	"crypto/subtle"
+
+	"chain/crypto/ed25519/internal/edwards25519"
+)
+
+type (
+	// Point is a point on the ed25519 curve.
+	Point edwards25519.ExtendedGroupElement
+
+	// EncodedPoint is a serialized representation of a point.
+	EncodedPoint [32]byte
```

Comment:
Removed `EncodedPoint` for now, but I suspect we might want to add it back - not for the sake of this package, which barely uses it, but because the CA code will be full of a few different kinds of `[32]byte` values, and it may make things clearer to name them distinctly.